src/MasterSite.php

```diff
@@ -733,6 +734,19 @@ public function view_meta_box_search( $post ) {
 		<?php
 	}
 
+	/**
+	 * Applies filters for list of users in dropdown
+	 *
+	 * @param null|Array $args The filter options and values.
+	 */
+	public function filter_authors( $args ) {
+		if ( isset( $args['who'] ) ) {
+			$args['role__in'] = [ 'administrator', 'author', 'campaigner', 'contributor', 'editor', 'subscriber' ];
```

Comment:
The JIRA ticket won't have a details about showing the `subscriber` & `contributor` user in dropdown. not sure do we need confirmation on it? 🤷🏽‍♂️

Reply:
We can safely skip `subscriber`, since we don't actively use that role and they don't have any backend permissions anyway.